old_tests/ut_expectations/ut.expect.not_to_be_null.sql

```diff
@@ -1,19 +0,0 @@
-@@lib/RunTest.sql "ut_expectations/common/ut.expect.unuary_comparator.scalar.common.sql 'blob' 'to_blob(''abc'')' 'not_to_be_null' 'ut_utils.tr_success'"
```

Comment:
Should we also remove `ut.expect.unuary_comparator.scalar.common.sql` or is it still used by other tests?

Reply:
We  can't do it, It's still used for two test